.gitignore

```diff
@@ -29,3 +29,5 @@
 /.node_modules.ember-try/
 /bower.json.ember-try
 /package.json.ember-try
+
+.DS_Store
```

Comment:
Can you remove that? This should rather live in your global git-ignore config IMHO.

Reply:
Done